bin/filter/filter_service_std.py

```diff
@@ -21,35 +22,44 @@ def main():
 
     reader = csv.DictReader(sys.stdin)
     outnames = ['owner_org'] + [f for f in reader.fieldnames
-        if f not in REMOVE_COLUMNS and f != 'owner_org'
-        ] + ['performance']
+        if f not in REMOVE_COLUMNS and f != 'owner_org'] \
+               + ['performance'] + ['target_met']
     writer = csv.DictWriter(sys.stdout, outnames)
     writer.writeheader()
     for row in reader:
         for rem in REMOVE_COLUMNS:
             del row[rem]
-        num = 0
-        den = 0
-        try:
-            num, den = num + int(row['q1_performance_result']), den + int(row['q1_business_volume'])
-        except ValueError:
-            pass
-        try:
-            num, den = num + int(row['q2_performance_result']), den + int(row['q2_business_volume'])
-        except ValueError:
-            pass
-        try:
-            num, den = num + int(row['q3_performance_result']), den + int(row['q3_business_volume'])
-        except ValueError:
-            pass
+
+        num = int(row['volume_meeting_target']) if row['volume_meeting_target'] else 0
+        den = int(row['total_volume']) if row['total_volume'] else 0
+
+        # performance = volume_meeting_target / total_volume
+        if num <= 0 or den <=0:
+            row['performance'] = 0
```

Comment:
`Performance` in their DEP is listed as "Optional", as are the `volume_meeting_target` and `total_volume` with the descriptions that no value inputted specifies that no data is available.

So I think `performance` should be the same, if `volume_meeting_target` or `total_volume`  <=0 then `row['performance'] = None` so that the value is null in the DS.